example_advanced.py

```diff
@@ -1,8 +1,4 @@
-"""This file is used to create and compare two models on a particular dataset.
```

Comment:
this example spits out an error. Maybe it's my machine.  I loaded into pycharm, hit run and get error:

ValueError: Invalid parameter max_depth for estimator KNeighborsClassifier. Check the list of available parameters with `estimator.get_params().keys()`.